app/templates/ui/app/detail/detail-no-esri.html

```diff
@@ -1,4 +1,18 @@
 <div class="row detail">
-  <!-- display JSON data -->
-  <json-explorer data="ctrl.doc"></json-explorer>
+	<tabset> <tab heading="XML" active="true">
+	<div>
+		<h1>XML</h1>
+		<pre class="prettyprint lang-xml">{{ctrl.model.tabs[0].content}}</pre>
```

Comment:
Interesting. Is that prettyprint class pointing to some CSS? I was also wondering if this looks better than highlightjs, since that is already loaded as dependency..